libs/designer/src/lib/core/actions/bjsworkflow/configuretemplate.ts

```diff
@@ -622,16 +622,20 @@ export const getWorkflowsWithDefinitions = async (
     return workflows;
   }
 
+  const allWorkflowKeys = Object.keys(workflows);
   const allWorkflows = Object.values(workflows);
+
   const promises = allWorkflows.map((workflow) =>
     getWorkflowDefinitionForStandard(workflow.manifest?.metadata?.workflowSourceId as string)
   );
   const allWorkflowsData = (await Promise.all(promises)).reduce((result: Record<string, Partial<WorkflowTemplateData>>, data, index) => {
     const { kind, definition: workflowDefinition } = data;
     if (workflowDefinition) {
+      const workflowId = allWorkflowKeys[index];
```

Comment:
does the id still contains the name of the actual resource and id field inside manifest is the newly changed one?